caluma/extensions/settings.py

```diff
@@ -123,7 +123,10 @@
     "intern-gesuchsart-intern-antrag-periodika",
 )
 
-settings.EARLY_CAREER_AWARD_FORM_SLUG = "nachwuchspreis-online"
+settings.EARLY_CAREER_AWARD_FORM_SLUGS = [
+    "early-career-award",  # prod
+    "eca-early-career-award",  # staging
+]
```

Comment:
If one is prod and the other is stating, couldn't we just use the often-copy-and-pasted `default()` settings function instead?